charts/kube-master/templates/api.yaml

```
@@ -121,6 +121,16 @@ spec:
             items:
               - key: local-kubeconfig
                 path: kubeconfig
+        {{- if ne .Values.audit "" }}
```

Can be simplified to
```suggestion
        {{- if Values.audit }}
```